tutorial_devops/consuming_conan_center.rst

```diff
@@ -0,0 +1,52 @@
+- The changes in recipes might also unpexectedly cause bugs and other issues. Recipes and package updates might bump dependencies versions to use the latest, forcing a continuous update over all dependencies versions. That can cause version conflicts if your project doesn't bump dependencies versions at the same pace.
+- Fixes to ConanCenter recipes can take a while to be merged (weeks), because there are too many contributions to be reviewed. If you rely on a fix that you contributed and it is urgent for your production system, you might need to wait.
+- Binary configurations can be discontinued in ConanCenter, when new compiler versions are released and ConanCenter starts to build them, it can drop some of the older versions.
+- Outages, maintenance windows and other service interruptions can happen, ranging from a few minutes to days. The infrastructure to host central repositories is typically large and complex, and relies on cloud providers that sometimes have their own issues, networks, CDN, etc.
+- Security issues. Even if the process to add packages to ConanCenter is very strict, with human reviews for every package and many automated checks and processes, eventually all systems are subject to attempts of supply chain attacks.
+
+Most of these reasons are not unique to Conan, but to every other package manager for other programming languages. The general devops known good practice is that you shouldn't rely on central repositories on production systems, but host your own copy of the packages that you need for production.
+
+This is the main idea: **host your own copy of the packages you need for production in your own server**. This could be done by just downloading binaries from ConanCenter and then uploading them to your own server. However, it is much better to fully own the complete supply chain and create the binaries in your own CI systems. So the recommended flow to use ConanCenter packages in production would be:
+
+- Create a fork of the ConanCenter Github repository: https://github.com/conan-io/conan-center-index
+- Create a list of the packages and versions you need for your projects. This list can be added to the fork too, and maintained there (packages can be added and removed with PRs when the teams need them).
+- Create a script that first ``conan export`` all the packages in your list, then ``conan create --build=missing`` them. Do not add ``user/channel`` to these packages, it is way simpler to use them as ``zlib/1.2.13`` without user-channel.
+- Upload your build packages to your own server, that you use in production, instead of ConanCenter.
+
```

Comment:
It would be amazing to have a diagram here for users that prefer those

```mermaid
flowchart LR
    id1[(conan-center-index)] <-->|fast forward pull or merge rebase| id2[(local-fork)]
    id2 --> d[[Currated List of Packages]]
    d --> e[/conan export name and version/]
    subgraph internal build proccess
    e --> f{conan create build missing}
    f --> g[conan upload list]
    end
```

Reply:
This is part of the detailed implementation, I prefer to wait a bit for this.